.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,95 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide us with the information requested in this bug report. 
+        Without these details, we won't be able to fully evaluate this issue. 
+        Bug reports lacking detail, or for any other reason than to report a bug, may be closed without action.
+
+        While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.
+        - If you have a support request or custom code related question please follow one of the steps below:
+        - Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
+        - If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
+        - Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)
+        - To get help on custom code questions go to the [WooCommerce Community Slack](https://woocommerce.com/community-slack/) and visit the `#developers` channel. 
+
+        Make sure to look through the existing issues to see whether your bug has already been submitted.
+        Feel free to contribute to any existing issues.
+        Search tip: You can filter our issues using [our labels](https://github.com/woocommerce/woocommerce/labels).
+        Search tip: Make use of [GitHub's search syntax to refine your search](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests).
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm these before submitting the issue.
+      options:
+        - label: I have carried out troubleshooting steps and I believe I have found a bug.
+        - label: I have searched for similar bugs in both open and closed issues and cannot find a duplicate.
+    validations:
+      required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      placeholder: |
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual behavior
+      placeholder: |
+        A clear and concise description of what actually happens. Please be as descriptive as possible;
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: Attach screenshot(s) or recording(s) directly by dragging & dropping.
```

Comment:
I will check if media can be included in a `textarea` or if this needs to be `markdown` instead. Might consider changing other `textarea` in the templates to `markdown`